src/runaqa.ts

```diff
@@ -233,10 +233,10 @@ async function getAqaTestsRepo(aqatestsRepo: string): Promise<void> {
   process.chdir('aqa-tests')
 }
 
-function getAqaSystemTestsRepo(aqasystemtestsRepo: string) {
+function getAqaSystemTestsRepo(aqasystemtestsRepo: string): void {
   const repoBranch = parseRepoBranch(aqasystemtestsRepo)
-  process.env.ADOPTOPENJDK_SYSTEMTEST_REPO = repoBranch[0];
-  process.env.ADOPTOPENJDK_SYSTEMTEST_BRANCH = repoBranch[1];
+  process.env.ADOPTOPENJDK_SYSTEMTEST_REPO = repoBranch[0]
```

Comment:
This probably won't work. env.ADOPTOPENJDK_SYSTEMTEST_REPO is the whole git url. See #115

Reply:
#115 can be done in a separate PR, which only affects system tests. Talked with @renfeiw we can let this in first.